src/util/saveArticles.ts

```diff
@@ -429,5 +400,7 @@ export async function saveArticles(zimCreator: ZimCreator, downloader: Downloade
   return {
     jsModuleDependencies,
     cssModuleDependencies,
+    jsMobileModuleDependencies,
+    cssMobileModuleDependencies,
```

Comment:
`jsModuleDependencies` and `cssModuleDependencies` should be updated, please don't create other variables to return.

Reply:
Done, I concated this data to the **jsDependenciesList** and **styleDependenciesList** in src/Downloader.ts.